config.yaml

```diff
@@ -77,3 +77,10 @@ options:
       A space-separated list of virtual IPs that the charm can utilize in conjunction with the
       HA Cluster subordinate charm. This option is mutually exclusive with the `ha-cluster-dns`
       option.
+  x-stream-protocol-version:
+    type: string
+    default: "v5.channel.k8s.io, v4.channel.k8s.io, v3.channel.k8s.io, v2.channel.k8s.io, channel.k8s.io"
```

Comment:
@addyess Should we default to `""` instead and allow the user to provide their own header values via charm config?